docs/data/sdks/marketing-analytics-browser/migration.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Browser SDK 2.0 Migration Guide
+description: Use this guide to easily migrate from Amplitude Marketing Analytics Browser to the Browser SDK 2.0.
+---
+
+Amplitude Browser SDK 2.0 (`@amplitude/analytics-browser`) features default event tracking, improved marketing attribution tracking, simplified interface and a lighter weight package.
+
+### Terminology
+
+* `@amplitude/analytics-browser@1`: Browser SDK 1.0
+* `@amplitude/analytics-browser@2`: Marketing Analytics Browser SDK 1.0
+
+### Dependency
+
+For snippet installation, update your project's [snippet loader](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader).
+
+For Node projects, update your dependency list in package.json.
+
+```diff
+{
+  "dependencies": {
+-     "@amplitude/marketing-analytics-browser": "^1"
++     "@amplitude/analytics-browser": "^2"
+  }
+}
+```
+
+### Default events tracking
+
+Starting Browser SDK 2.0, default tracking is enabled by default. Default tracking is implicit tracking performed by Amplitude on your behalf, and includes page views, sessions, file downloads, form interactions, and marketing attribution.
+
+To opt out of default tracking, set `options.defaultTracking` to `false`.
+
+```ts
+amplitude.init(API_KEY, undefined, {
+  defaultTracking: false,
+});
+```
+
+Additionally, you can pick and choose which events you want tracked by Amplitude. For example, if you only want default tracking for marketing attribution and page views, you can use the code below.
+
+```ts
+amplitude.init(API_KEY, undefined, {
+  defaultTracking: {
+    attribution: true,
+    pageViews: true,
+    sessions: false,
+    fileDownload: false,
+    formInteraction: false,
+  },
+});
+```
+
+### Deprecates user agent client-side parsing
+
+In Browser 2.0, we removes functionality to parse user agent that populates `event.os_name`, `event.os_version`, `event.device_model`, `event.device_manufacturer` on the client side. This is replaced with user agent server-side parsing which offered by Amplitude ingestion endpoints. This breaking changes might affect your chart analystic related to the user agent properties. However, we provided a [user agnet enrichment plugin](https://www.npmjs.com/package/@amplitude/plugin-user-agent-enrichment-browser) to enrich the user agent info to keep it backward compatible. Please check [here](https://github.com/amplitude/Amplitude-TypeScript/tree/v1.x/packages/plugin-user-agent-enrichment-browser) for more details.
```

Comment:
Not sure if it would be too tedious, but would be nice to make the shared migration parts a "include" to make future maintenance easier.